machinelearning/horovod/README.md

```diff
@@ -0,0 +1,127 @@
+# Horovod
+
+Horovod is a distributed training framework for TensorFlow, Keras, and PyTorch. The goal of Horovod is to make distributed Deep Learning fast and easy to use.
+
+This is a port of the Dockerfile maintained at https://github.com/uber/horovod
+
+## Instructions
+
+A difference between the running the singularity container (vs the docker one) is that mpi runs outside of singularity thus you do not have to start the container instance on each host as detailed in https://github.com/uber/horovod/blob/master/docs/docker.md .
+
```

Comment:
Same here, add the hyperlink -> [https://github.com/uber/horovod/blob/master/docs/docker.md](https://github.com/uber/horovod/blob/master/docs/docker.md)